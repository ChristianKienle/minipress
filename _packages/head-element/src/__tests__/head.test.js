// @ts-check

const Head = require('./../index')

describe('Head', () => {
  it('renders Title', () => {
    expect(new Head().title('hello')
      .renderToString()).toMatchSnapshot()
  })

  it('renders Description', () => {
    expect(new Head().description('my site')
      .renderToString()).toMatchSnapshot()
  })

  it('renders Meta', () => {
    expect(new Head().meta('Test', 'value')
      .renderToString()).toMatchSnapshot()
  })
})
