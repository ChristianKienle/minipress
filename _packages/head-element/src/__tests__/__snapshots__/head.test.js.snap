// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Head renders Description 1`] = `
"<title></title>
<meta name=\\"Description\\" content=\\"my site\\">"
`;

exports[`Head renders Meta 1`] = `
"<title></title>
<meta name=\\"Test\\" content=\\"value\\">"
`;

exports[`Head renders Title 1`] = `"<title>hello</title>"`;
